swarm/pss/prox_test.go

```diff
@@ -213,36 +187,50 @@ func (d *testData) init(msgCount int) error {
 // nodes Y and Z will be considered required recipients of the msg,
 // whereas nodes X, Y and Z will be allowed recipients.
 func TestProxNetwork(t *testing.T) {
-	t.Run("16/16/15", testProxNetwork)
+	t.Run("short", func(t *testing.T) {
+		testProxNetwork(t, 16, 16, 16)
+	})
 }
 
 // params in run name: nodes/msgs
 func TestProxNetworkLong(t *testing.T) {
 	if !*longrunning {
 		t.Skip("run with --longrunning flag to run extensive network tests")
 	}
-	t.Run("8/100/30", testProxNetwork)
-	t.Run("16/100/30", testProxNetwork)
-	t.Run("32/100/60", testProxNetwork)
-	t.Run("64/100/60", testProxNetwork)
-	t.Run("128/100/120", testProxNetwork)
+	t.Run("longrunning1", func(t *testing.T) {
```

Comment:
please keep the more informative names like: `16 nodes, 100 msgs`

Reply:
IMHO, the current implementation is easier to understand. I call the same function 5 times with 10 lines of meaningful code. You understand it from the first glance. With your example, you need more effort to understand what all these functions do, and how they fit together.